.github/workflows/cno-ci.yml

```diff
@@ -0,0 +1,131 @@
+name: ovn-cno-ci
+
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+
+env:
+  GO_VERSION: 1.13.4
+  K8S_VERSION: v1.18.2
+  KIND_CLUSTER_NAME: ovn
+  BUILD_CNO: true
+  BUILD_OVN: true
+
+jobs:
+  k8s:
+    name: Build k8s
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Set up environment
+        run: |
+          export GOPATH=$(go env GOPATH)
+          echo "::set-env name=GOPATH::$GOPATH"
+          export PATH=$GOPATH/bin:$PATH
+          echo "::add-path::$GOPATH/bin"
+
+      - name: Cache Kubernetes
+        id: cache-k8s
+        if: github.event_name == 'push' || github.event_name == 'pull_request'
+        uses: actions/cache@v1
```

Comment:
@nerdalert can you update to cache@v2? We recently did that for the other jobs.

Reply:
Done ty.